packages/client/components/Dashboard.tsx

```diff
@@ -88,7 +89,7 @@ const Dashboard = (props: Props) => {
           </Switch>
         </DashMain>
       </DashPanel>
-      <StartMeetingFAB />
+      {isDesktop ? <StaticStartMeetingFAB /> : <StartMeetingFAB />}
```

Comment:
- Personally, I prefer the button aligned with the user's avatar on the top bar for wide viewports 
- without the autocollapse we're giving up a lot of screen real estate
- straying from the 8px grid & using a 28 pixel offset looks off.
<img width="501" alt="Screen Shot 2020-01-22 at 1 29 57 PM" src="https://user-images.githubusercontent.com/5514175/72922655-6e2bd880-3d1b-11ea-8b5a-7a4130b7a35a.png">

Reply:
- re: aligned with top-right avatar, I agree but it’s going to take some implementation. First the top bar should max out at the same width as the main area. I want to work on this in a quick commit. Second, the button position should be based on the right side of the main area, not the whole viewport, not over the right sidebar. We need to structure this at the root level of the dashboard, that any right sidebar element pushes both the main area and the FAB. When a right sidebar is open on mobile, we get a scrim to close, and won’t see the FAB. I’m making an issue of the second part of this.
- re: I looked at it not collapsing (over tasks, etc.) and I think it’s safer than the button being lost and/or confused for a help widget; I propose keeping it static for at least a little while so regular users become aware of it, and also I’m on the hunt for a better global meeting icon.
- re: 28px offset, yeah, the intent here and overall is to give the interface more space to breathe when there’s room for it. The 28px wasn’t arbitrary, just borrowed, as that’s half the FAB height and usually where it sits when it’s halfway over a bottom bar. If I stuck to the 8px grid I’d probably opt for 24px or 32px at the larger breakpoint.